activerecord/lib/active_record/connection_adapters/abstract_adapter.rb

```diff
@@ -77,7 +77,14 @@ class AbstractAdapter
       SIMPLE_INT = /\A\d+\z/
 
       attr_accessor :visitor, :pool
-      attr_reader :schema_cache, :owner, :logger, :prepared_statements, :lock
+
+      attr_reader :schema_cache,
+                  :owner,
+                  :logger,
+                  :prepared_statements,
+                  :lock,
+                  :advisory_locks
```

Comment:
Should this be `:advisory_locks_enabled`?

Reply:
I think it is matching `prepared_statements`. I'm fine with either one.

Reply:
The instance variable is named `@advisory_locks_enabled`.

Reply:
@georgeclaghorn  Oops good catch. Thank you for fixing @kamipo :+1: